landing-page/content/common/join.md

```diff
@@ -64,20 +64,3 @@ Iceberg has four mailing lists:
     - [Archive](https://lists.apache.org/list.html?issues@iceberg.apache.org)
 * **Private**: <private@iceberg.apache.org> -- private list for the PMC to discuss sensitive issues related to the health of the project
     - [Archive](https://lists.apache.org/list.html?private@iceberg.apache.org)
-
-
-## Setting up IDE and Code Style
```

Comment:
@samredai, did this page get renamed? I think it should be `community.md` and it shows up at that URL (https://iceberg.apache.org/community/)